### PR TITLE
Create guarda.rb

### PR DESCRIPTION
**Pull request by jonathanstanley**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free. 
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [ ] `brew audit --new-cask <cask>` worked successfully. (exception: has its own website; the developers just use GitHub for hosting the binaries.  for comparison, their [android app has 100k+ installs](https://play.google.com/store/apps/details?id=com.crypto.multiwallet&hl=en_US&gl=US))
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/guarda.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask "guarda" do
+  version "1.0.20"
+  sha256 "69e88c2ccb05ec2f8004045f0d0940ae578b51992975e6b8bdbda78e0e95e044"
+
+  url "https://github.com/guardaco/guarda-desktop-releases/releases/download/v#{version}/Guarda-#{version}.dmg", verified: "github.com/guardaco/"
```
Comment (neutric) [inline Casks/guarda.rb:5]: The `verified` stanza should be `verified: "github.com/guardaco/guarda-desktop-releases/"` and it belongs on its own line.

File: Casks/guarda.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  name "guarda"
```
Comment (neutric) [inline Casks/guarda.rb:6]: `name "Guarda"`

**Review by neutric — COMMENTED** [review]
Thanks for your contribution @jonathanstanley! Please take my comments with a grain of salt as I am not a maintainer.

File: Casks/guarda.rb (hunk 2 above)
Comment (jonathanstanley) [reply Casks/guarda.rb]: fixed

**Comment by miccal** [comment]
Thank you for your submission @jonathanstanley, but we cannot accept it at this time because it does not meet our [minimum notability requirements](https://docs.brew.sh/Acceptable-Casks#rejected-casks):

> + The Cask is for an app that is too obscure. Examples:
>   + An app from a code repository that is not notable enough (under 30 forks, 30 watchers, 75 stars).

Please consider hosting this [in your own Tap](https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap) and if it gets notable enough we would then be happy to consider adding this to `homebrew-cask`.

**Pull request closed without merging**